initial_setup/gui/gui.py

```diff
@@ -9,9 +9,17 @@
 _ = lambda x: gettext.ldgettext("initial-setup", x)
 N_ = lambda x: x
 
+def get_quit_message():
+    if eula_available():
+        return N_("Are you sure you want to quit the configuration process?\n"
+                  "You might end up with an unusable system if you do. Unless the "
+                  "License agreement is accepted, the system will be rebooted.")
+    else:
+        return N_("Are you sure you want to quit the configuration process?\n"
+                  "You might end up with unusable system if you do.")
+
```

Comment:
Is there a reason to have this as top level function instead of method inside of `InitialSetupQuitDialog` class?